src/extensions/default/BrambleUrlCodeHints/camera/index.js

```diff
@@ -8,11 +8,11 @@ define(function (require, exports, module) {
 
     var CommandManager = brackets.getModule("command/CommandManager");
     var Commands       = brackets.getModule("command/Commands");
+    var fs             = brackets.getModule("fileSystemImpl");
```

Comment:
yeah basically! We seem to be using this in most of our bramble extensions so thought we should use it here too. Is there something else we should use that's better?